internal/generate/testdata/non-standard-layout/expected-catalog/olm-catalog/memcached-operator/0.0.4/memcached-operator.v0.0.4.clusterserviceversion.yaml

```diff
@@ -30,8 +30,6 @@ spec:
       - description: Size is the size of the memcached deployment
         displayName: Size
         path: size
-        x-descriptors:
-        - urn:alm:descriptor:com.tectonic.ui:podCount
       statusDescriptors:
       - description: Nodes are the names of the memcached pods
```

Comment:
We have no longer `<dir>/<tag>/csv.<tag>`. I think we need to update the mocktestdata accordingly to the current behaviour/structure on the legacy layout.

Reply:
That deprecation was recently [un-deprecated](#2944).